player/command.c

```diff
@@ -2583,6 +2583,39 @@ skip_warn: ;
     return M_PROPERTY_NOT_IMPLEMENTED;
 }
 
+static bool floats_equal(float x, float y) {
+    float TOLERANCE = 0.001;
+    float difference = x > y ? x - y : y - x;
+    return difference <= TOLERANCE;
+}
+
+static int mp_property_video_aspect_override(void *ctx, struct m_property *prop,
+                                             int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+    if (action == M_PROPERTY_PRINT) {
+        float aspect_ratio;
+        mp_property_generic_option(mpctx, prop, M_PROPERTY_GET, &aspect_ratio);
+
+        if (floats_equal(aspect_ratio, 2.35F / 1.0F))
+            *(char **)arg = talloc_asprintf(NULL, "2.35:1");
+        else if (floats_equal(aspect_ratio, 16.0F / 9.0F))
+            *(char **)arg = talloc_asprintf(NULL, "16:9");
+        else if (floats_equal(aspect_ratio, 16.0F / 10.0F))
+            *(char **)arg = talloc_asprintf(NULL, "16:10");
+        else if (floats_equal(aspect_ratio, 4.0F / 3.0F))
+            *(char **)arg = talloc_asprintf(NULL, "4:3");
+        else if (floats_equal(aspect_ratio, -1.0F))
+            // FIXME: `Original` not localized
+            *(char **)arg = talloc_asprintf(NULL, "Original");
```

Comment:
There's no localization in mpv currently. There used to, years ago, but it was incrementally removed due to bitrot and because there was no interest. We generally assume people who want localization use frontends anyway, or do not care about what little text mpv displays in pseudo-GUI mode.